### PR TITLE
feat(checkout): CHECKOUT-7264 Update our Dev Docs with the new Line Items Overrides

### PR DESCRIPTION
**Pull request by valentindellangela**
# Working on https://bigcommercecloud.atlassian.net/browse/CHECKOUT-6950

## What changed?
We added 2 new fields: `weight` and `dimensions` to the following endpoint:
* POST V3/Carts
* POST V3/Carts/xxx/Item
* PUT V3/Carts/xxx/Item

This is to allow a merchant to override those values using our Server To Server API.

## Proof?
No errors in Swagger IO and the fields appear properly in the requests and responses of those API
image

## Anything else?

- [x] The work is gated under an experiment `CHECKOUT-6950.enable_item_override` so we should likely postpone the release of the documentation until we have fully ramped up the experiment and haven't found any bug.

This PR can be merged anytime as the feature has been fully ramped up.

ping @bc-charlesho @bc-traciporter

### CODE REVIEW

**Review by bc-mackxu — APPROVED** [review]
LGTM

**Comment by valentindellangela** [comment]
Hi @bc-traciporter / @bigcommerce/dev-docs-collaborators this feature has been fully ramped up so we can now merge this PR. 🙇

**Comment by bc-traciporter** [comment]
Hi @valentindellangela, Our docs are going into freeze mode while we migrate. Let me talk to @slsriehl to see if she can merge and publish these changes.

File: reference/carts.v3.yml [hunk 1]
```diff
@@ -2793,6 +2825,25 @@ components:
               type: string
               description: The item’s product name.
               example: T-Shirt
+            weight:
+              type: number
+              description: If the item has a custom weight, it will be returned here.
```
Comment (bc-traciporter) [inline reference/carts.v3.yml:2830]:
```suggestion
              description: The weight is displayed here if the item has a custom dimension.
```

File: reference/carts.v3.yml [hunk 2]
```diff
@@ -2793,6 +2825,25 @@ components:
...
+              example: 1.2
+            dimensions:
+              type: object
+              properties:
+                height:
+                  type: number
+                  description: If the item has a custom dimension, the height will be returned here.
```
Comment (bc-traciporter) [inline reference/carts.v3.yml:2837]:
```suggestion
                  description: The height is displayed here if the item has a custom dimension.
```

File: reference/carts.v3.yml [hunk 3]
```diff
@@ -2793,6 +2825,25 @@ components:
...
+                  example: 2.0
+                width:
+                  type: number
+                  description: If the item has a custom dimension, the width will be returned here.
```
Comment (bc-traciporter) [inline reference/carts.v3.yml:2841]:
```suggestion
                  description: The width is displayed here if the item has a custom dimension.
```

File: reference/carts.v3.yml [hunk 4]
```diff
@@ -2793,6 +2825,25 @@ components:
...
+                  example: 2.1
+                depth:
+                  type: number
+                  description: If the item has a custom dimension, the depth will be returned here.
```
Comment (bc-traciporter) [inline reference/carts.v3.yml:2845]:
```suggestion
                  description: The depth is displayed here if the item has a custom dimension.
```

**Comment by slsriehl** [comment]
[DEVDOCS-4778]

**Comment by valentindellangela** [comment]
> Hi @valentindellangela, Our docs are going into freeze mode while we migrate. Let me talk to @slsriehl to see if she can merge and publish these changes.

Hi @bc-traciporter , we are not in a hurry to merge this one so it can wait until the freeze period is over.
Thanks

**Comment by slsriehl** [comment]
Thanks @valentindellangela

**Comment by valentindellangela** [comment]
@bc-traciporter @slsriehl  Are we still on code freeze?
I think my other api-specs PR got merged a few days ago.

**Comment by valentindellangela** [comment]
Thank you 🙇

**Pull request merged**